docs/07_how-to/09_fdc3.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: 'How to set up interoperability'
+sidebar_label: 'Interoperability/FDC3'
+id: ht-fdc3
+keywords: [fdc3, interoperability]
+tags:
+    - fdc3
+    - interoperability
+
+---
+
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+A Genesis app can use the FDC3 standard for interoperability between front-end applications. 
+
+So, if you have apps that provide services such as charting or news, then you can set up your app so that can share messages with those apps at the front end.
+
+For example, a user could click on a row in a grid of data in your Genesis application and then view view news or a chart on that instrument in a separate app window.
```

Comment:
"view view"

we should give context of what your clicking on not just " a grid" :

"For example, a user could click on a row in a grid which contains instrument data in your Genesis application, and other apps listening could update the news ticker or perhaps a chart to view that same instrument"

Reply:
"and vice versa"